src/Config/SimpleConfig.php

```diff
@@ -22,6 +22,7 @@
     private $include_paths;
     private $entry_points;
     private $asset_files;
+    private $excluded_files;
```

Comment:
`$excluded_imports`

Reply:
yeah, not sure about that... You have to give it actual files. `"jquery"` will not work for instance. This is because I do not have access to the resolvers.